wordpress-mobile/WordPress-iOS
# Remote auto-save unconfirmed published posts

Fixes #12322

## To test

While offline:
- Create a post
- Tap publish
- Tap cancel
- Go back online and check that a revision was created (aka the post was autosaved)

## Additional information

Issue #12322 mentioned that an editor crash was another way to trigger an `.autoSave` state. Currently, this *does not happen* since no post upload attempt is made.

Anyway, in the case of a crash, the content will be locally saved.

Update release notes:
